src/main/java/net/minecraftforge/fml/hooks/BasicEventHooks.java

```diff
@@ -68,15 +68,15 @@ public static void firePlayerSmeltedEvent(PlayerEntity player, ItemStack smelted
         MinecraftForge.EVENT_BUS.post(new PlayerEvent.ItemSmeltedEvent(player, smelted));
     }
 
-    public static void onRenderTickStart(float timer)
+    public static void onRenderTickStart(float timer, boolean doRenderWorld)
```

Comment:
All of these need defaults, we can't break binary compatibility after a recommended build.